### ISSUE TITLE
Unchecked checkboxes should have an icon

### ISSUE THREAD

**Issue by gsbhasin123** [muckSponge/MaterialFox#232]
**Sanity checks (must complete)**
- [x] I have read and followed the installation instructions in the README
- [x] I have not modified the userChrome.css file
- [x] I have tested the latest release for my Firefox version, or commit on master/beta branch (beta is for Firefox Beta only)

Manjaro KDE with the latest Firefox 76.
An unchecked checkbox looks like this:
![image](https://user-images.githubusercontent.com/48339289/81441589-e41b4a80-9126-11ea-86d7-a0e0b1ea3919.png)
It should be an empty box a different color from the menu to distinguish that it is a checkbox.
Until this is fixed (i don't know how to fix it), I have renamed the popup folder to popupstub so that the css will not find it.

**Comment by muckSponge** [comment]
I was trying to replicate [this behaviour](https://material.io/components/menus):

![image](https://lh3.googleusercontent.com/gwoLF43cfIgeBckW-jxaN7xI3y6T3PqRnGNO7i8Kf3Mbouqh0vzeCgzm1JPLvvnrWkhh0Js2epA0c85dmv-dXREcvrsujg69R3dv=w1064-v0)

Material Design spec doesn't really have empty checkboxes in menus, but we don't have to follow it to the letter if people would prefer. I guess there's also no case in the design spec where a single checkbox is used (only what are essentially radios with more than one option), so by that logic they could have checkboxes.

**Comment by codic12** [comment]
Yeah, it looks good there, but looks pretty out of place in the MaterialFox context menu. Also, how can i decrease the padding?
